llvm/docs/CommandGuide/llvm-exegesis.rst

```diff
@@ -425,6 +431,12 @@ OPTIONS
   features such as memory annotations but is currently restricted to X86-64
   on Linux.
 
+.. option:: --benchmark-repeat-count=<repeat-count>
+
+  This option enables specifying the number of times to repeat the benchmark
+  when performing latency benchmarks. By default, llvm-exegesis will repeat
+  a latency measurement 30 times and then aggregate the results.
```

Comment:
Let's avoid referring to a specific value because  people should not really depend on the default value and it's likely to get out of sync if we change the value.

Reply:
Changed this out for less-specific language that should better encompass what we're going for here.